src/zen_ecp.c

```diff
@@ -779,6 +806,8 @@ int luaopen_ecp(lua_State *L) {
 		{"mul", ecp_mul},
 		{"validate", ecp_validate},
 		{"prime", ecp_prime},
+		{"rhs", ecp_rhs},
+		{"zcash_export", ecp_zcash_export},
```

Comment:
this can be simply a `:zcash()` method of ECP (since it needs always ECP as 1st arg).

If there are both import and export functions then we use `:from_zcash()` and `:to_zcash()` as method names in zenroom.